toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -55,6 +55,9 @@ const (
 	// CmdlineSELinuxEnforcingArg is the arg required for forcing SELinux to be in enforcing mode.
 	CmdlineSELinuxEnforcingArg = "enforcing=1"
 
+	// CmdlineSELinuxPermissiveArg is the arg for SELinux to be in force-permissive mode.
+	CmdlineSELinuxPermissiveArg = "enforcing=0"
```

Comment:
Since this change is outside the MIC directories, it will need to be approved by shiproom. I recommend making the change that shiproom needs to review as small as possible. So, it might be good to handle the SELinux stuff as a separate PR.